### DISCUSSION TITLE
Codemagic Build Error "You must let us know whether your app uses any Foreground Service permissions.", "status": "PERMISSION_DENIED in Android"

### DISCUSSION THREAD

**Discussion by NiladriPadhy** [codemagic-ci-cd/codemagic-docs#2719] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question
.NET8 MAUI for Android

### Steps to reproduce

Preparing build machine Done

Fetching app sources Done
Installing SDKs Done
Set up code signing identities Done
Installing dependencies Done
Building Android Done
Publishing Getting Error like below
Publishing artifact com.readywire.attendance-Signed-artifacts.aab

Publishing com.readywire.attendance-Signed-artifacts.aab to Google Play
Published com.readywire.attendance-Signed.aab to track internal
Google Play failed to upload artefacts. You must let us know whether your app uses any Foreground Service permissions.: {
    "error": {
        "code": 403,
        "message": "You must let us know whether your app uses any Foreground Service permissions.",
        "status": "PERMISSION_DENIED"
    }
}

Build failed :|

Publishing failed :|
Google Play failed to upload artefacts. You must let us know whether your app uses any Foreground Service permissions.: {
    "error": {
        "code": 403,
        "message": "You must let us know whether your app uses any Foreground Service permissions.",
        "status": "PERMISSION_DENIED"
    }
}

### Expected results

Build and Published successfuly

### Actual results

Publishing failed :| Google Play failed to upload artefacts. You must let us know whether your app uses any Foreground Service permissions.: { "error": { "code": 403, "message": "You must let us know whether your app uses any Foreground Service permissions.", "status": "PERMISSION_DENIED" } }

### Build id (optional)

66da2713863a11458378b0e2

**Comment by himesh-cm** [comment]
Hey @NiladriPadhy ,
The error you’re encountering is related to Google Play’s updated requirements for apps using Foreground Services. 
Please refer to the documentation at https://developer.android.com/about/versions/14/behavior-changes-14#fgs-types and the helpful SO thread at https://stackoverflow.com/questions/76943771/foreground-service-crashing-on-android-14.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
